roles/network_plugin/weave/templates/weave-kube.j2

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
This should be weave-kube.yaml.j2 (resulting extension should be in the filename with .j2 at the end)